From 384bcefafb8c2a23dfff135e18008a0ee8405706 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 27 Sep 2012 10:22:26 +0100
Subject: [PATCH 108/118] RHEL 6: In mount-local docs, change refs to
 libguestfs 1.18 -> RHEL 6.4.

We backported the libguestfs 1.18 / mount-local functionality
into the libguestfs 1.16 codebase for RHEL 6.4.
---
 src/guestfs.pod | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/guestfs.pod b/src/guestfs.pod
index 9f67167..6c78dab 100644
--- a/src/guestfs.pod
+++ b/src/guestfs.pod
@@ -228,7 +228,7 @@ language bindings turn these errors into exceptions).
 File writes are affected by the per-handle umask, set by calling
 L</guestfs_umask> and defaulting to 022.  See L</UMASK>.
 
-Since libguestfs 1.18, it is possible to mount the libguestfs
+Since RHEL 6.4, it is possible to mount the libguestfs
 filesystem on a local directory, subject to some restrictions.  See
 L</MOUNT LOCAL> below.
 
@@ -559,7 +559,7 @@ device (I<not> the underlying encrypted block device).
 
 =head2 MOUNT LOCAL
 
-In libguestfs E<ge> 1.18, it is possible to mount the libguestfs
+In RHEL E<ge> 6.4, it is possible to mount the libguestfs
 filesystem on a local directory and access it using ordinary POSIX
 calls and programs.
 
@@ -590,8 +590,8 @@ L</guestfs_mount_local_run>.
 
 =head3 MOUNT LOCAL COMPATIBILITY
 
-Since local mounting was only added in libguestfs 1.18, and may not
-be available even in these builds, you should consider writing code
+Since local mounting was only added in RHEL 6.4,
+you should consider writing code
 so that it doesn't depend on this feature, and can fall back to
 using libguestfs file system calls.
 
-- 
2.7.4

